tests/test_reply_body.js

```diff
@@ -91,3 +91,24 @@ test('unencodable object throws the expected error', t => {
 
   t.end()
 })
+
+test('reply with missing body defaults to empty', async t => {
+  const scope = nock('http://example.test')
+    .get('/')
+    .reply(204)
+
+  const { statusCode, body } = await got('http://example.test/')
+
+  t.is(statusCode, 204)
+  t.equal(body, '')
+  scope.done()
+})
+
+test('reply with missing status code throws an error', async t => {
```

Comment:
Thoughts on changing this behavior? I don't see a reason that a plain call to `.reply()` couldn't keep the default it had before. It's true that `.reply(() => [])` throws an error, though accepting empty call is actually useful shortcut, whereas accepting a function that returns `[]` doesn't seem useful in any way, when the function could be omitted instead.

Reply:
I've made this update to default to 200 again.